.golangci.yml

```diff
@@ -0,0 +1,13 @@
+linters-settings:
+  lll:
+    line-length: 141
```

Comment:
Wow, why 141 characters? :laughing:

Reply:
I refuse to use shorter error messages to keep the line length short :)

Reply:
Isn't there a way to split them in several lines, without too much hassle?